### PR TITLE
add correct labels to pandas_function_request.md

### PR DESCRIPTION
**Pull request by raybellwaves**
The should correct the labels added on a the "Request a Missing Pandas Function" template.

"? - Needs Triage" -> "Needs Triage" (https://github.com/rapidsai/cudf/issues?q=is%3Aopen+is%3Aissue+label%3A%22Needs+Triage%22)

and adds the "pandas" label (https://github.com/rapidsai/cudf/issues?q=is%3Aopen+is%3Aissue+label%3Apandas)

### CODE REVIEW

**Comment by raybellwaves** [comment]
Friendly ping @beckernick as I believe you wrote this code.

**Comment by vyasr** [comment]
@galipremsagar could you take a look at this? I think we actually probably want to use one of the new labels that you created for this purpose.

File: .github/ISSUE_TEMPLATE/pandas_function_request.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Request a Missing Pandas Function
 about: Request GPU support for a function executed on the CPU in pandas accelerator mode.
 title: "[FEA]"
-labels: "? - Needs Triage, feature request"
+labels: "Needs Triage, feature request, pandas"
```
Comment (galipremsagar) [inline .github/ISSUE_TEMPLATE/pandas_function_request.md:5]:
```suggestion
labels: "Needs Triage, feature request, cudf.pandas"
```

**Review by galipremsagar — APPROVED** [review]

**Review by AyodeAwe — APPROVED** [review]

**Comment by vyasr** [comment]
Thanks @raybellwaves!

**Pull request merged**